.changeset/forty-pumas-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+'@emotion/react': patch
+---
+
+expose \_\_unsafe_useEmotionCache from @emotion/react, see https://github.com/emotion-js/emotion/pull/2441
```

Comment:
```suggestion
Exposed `__unsafe_useEmotionCache` which can be used to access the current Emotion's cache in an easier way than before. Using this might break 0-config SSR and is not recommended to be used unless there you know what you are doing and you are OK with the mentioned downside.
```